package.json

```diff
@@ -70,6 +72,7 @@
     "babel-preset-airbnb": "^3.2.1",
     "chai": "^4.2.0",
     "clean-css": "^4.2.1",
+    "core-js": "^3.1.3",
```

Comment:
why is this needed?

Reply:
The commit message has the details: https://github.com/airbnb/react-dates/commit/15611832b01a2d298789af4c0049c96d2b794cfa

Reply:
The implication is that by using storybooks webpack config we have created this problem for ourselves. should that be reverted?

Reply:
I think there’s no real way out of using storybook’s webpack config. And just to be clear, this isn’t a happo issue - it just happened to trigger the issue.

Perhaps I’m misunderstanding something though, can you elaborate on what you mean should be reverted?

Reply:
why must we use their config, versus manually replicating the parts we need (since the linked issue is due to weird aliasing in it)?

Reply:
The alias isn’t there anymore, tried removing it. A theory I have is that they’re generating some code that ends up being bundled with webpack. At least that would explain why a `require('core-js')` wouldn’t resolve to the local copy of core-js (storybook/core/node_modules/core-js). If I have some time I’ll explore this theory and try to push a fix upstream.